### PR TITLE
Cold chain docs

### LINKED ISSUE #15: Cold chain documentation
Create documentation for the cold chain integration work, to be released with oms v1.5.0

Areas to cover:
- API integration
- Chart display
- Breach display
- Log display
- Notifications
- Upload of berlinger data
- Managing sensors

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #15 

Adds some documentation for the cold chain feature 🎉

### CODE REVIEW

File: content/docs/coldchain/introduction.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+The cold chain features within open mSupply allow you to:
+- View and manage temperature sensors
+- Import data from Berlinger Q-tag temperature loggers
+- View logs and breach data from sensors
+- Receive notifications of temperature breach events
+ 
+Sensors can be assigned to a storage location, which then allows you to associate temperature logs and breaches with specific stock lines. You can also view historical data to check if your cold chain equipment is operating correctly and to check if your stock might have been affected by temperature variations.
+
+If a temperature breach happens, then a notification is displayed on every page of open mSupply as a warning to check potentially affected stock. Once the breach is dismissed then the notification is no longer shown.
```
Comment (roxy-dao) [inline content/docs/coldchain/introduction.md:24]:
```suggestion
If a temperature breach happens, then a notification is displayed on every page of Open mSupply as a warning to check potentially affected stock. Once the breach is acknowledged then the notification is no longer shown.
```

Comment (roxy-dao) [reply content/docs/coldchain/introduction.md:24]: Also every place where open mSupply is used -> Open mSupply
and cold chain application -> Cold Chain Application (application?)

File: content/docs/coldchain/introduction.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+- Vaccines
+  - View sensor details
+  - Edit sensor location
+
+See the [user permissions](https://docs.msupply.org.nz/admin:managing_users#permissions_tabs) documentation for more details.
+
+## Cold chain app integration
+
+The [Cold Chain](/coldchain/introduction/) application is able to sync data to a server. The server can be an instance of mSupply or now, open mSupply.
```
Comment (roxy-dao) [inline content/docs/coldchain/introduction.md:44]:
```suggestion
The [Cold Chain](/coldchain/introduction/) application is able to sync data to a server. The server can be an instance of Open mSupply or mSupply.
```


Swap the order 😉

File: content/docs/coldchain/introduction.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+To configure the integration, you will only need to configure a user and then you can update the cold chain application.
+
+The user is a standard mSupply user, with the following configuration:
+- The default store assigned to the user is the store which the temperature data will be associated with 
+- The store will need to be part of the site that open mSupply is synchronising with. Have a look at the [Admin](/docs/administration/synchronisation/#viewing-the-synchronisation-settings) section to see which site omSupply is syncing with, and then the [Synchronisation](https://docs.msupply.org.nz/synchronisation:sync_sites#viewing_sync_sites) screen to check that your site includes the correct store.
```
Comment (roxy-dao) [inline content/docs/coldchain/introduction.md:50]: Open mSupply

right below this on line 51 too

File: content/docs/coldchain/monitoring.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+| Column              | Description                      |
+| :------------------ | :------------------------------- |
+|                     | An icon showing an alert if this breach is unacknowledged        |
+| **Status**          | The breach status can be `Acknowledged` or `Unacknowledged`     |
+| **Sensor name**     | Name of the Sensor               |
+| **Location**        | Current location assigned to the sensor |
+| **CCE**             | Name of the cold chain equipment item which the sensor is associated with   |
+| **Breach start**    | Date and time at which the breach started |
+| **Duration**     | The duration of the breach, if it has ended. If the breach has not ended then `Ongoing` is shown |
```
Comment (roxy-dao) [inline content/docs/coldchain/monitoring.md:68]: I didn't know it showed Ongoing 🤔 ... I do see the label though in om, but it's not being used 👀

File: content/docs/coldchain/monitoring.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+- Sensor name
+- Location
+- Date/Time Range
+- Breach type
+
+To add a filter to the page, choose the required filter from the drop down. 
+
+![Filter](/docs/coldchain/images/log_filter.png)
+
+1. The list can display a fixed number of sensors per page. On the bottom left corner, you can see how many sensors are currently displayed on your screen.
```
Comment (roxy-dao) [inline content/docs/coldchain/monitoring.md:124]:
```suggestion
1. The list can display a fixed number of logs per page. On the bottom left corner, you can see how many logs are currently displayed on your screen.
```

File: content/docs/coldchain/monitoring.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+- Location
+- Date/Time Range
+- Type
+- Acknowledged
+
+To add a filter to the page, choose the required filter from the drop down. 
+
+![Filter](/docs/coldchain/images/breach_filter.png)
+
+1. The list can display a fixed number of sensors per page. On the bottom left corner, you can see how many sensors are currently displayed on your screen.
```
Comment (roxy-dao) [inline content/docs/coldchain/monitoring.md:84]:
```suggestion
1. The list can display a fixed number of breaches per page. On the bottom left corner, you can see how many breaches are currently displayed on your screen.
```

File: content/docs/coldchain/notifications.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- The vaccine module has been enabled for the current store
+- The currently logged in user has permission to view temperature logs
+- One or more sensors have recorded a temperature breach
+- One or more of those breaches has not been acknowledged
+
+Here's how the bar appears, shown at the top of the dashboard page:
+
+![notification bar](/docs/coldchain/images/notification_bar.png)
+
+<div class="tip">If the store has the vaccine module enabled, and the user does not have permission to view logs, or some other error occurs, then you will see a notification when you first login or change stores: <p><img src="/docs/coldchain/images/notification_warning.png" alt="Notification permission warning" /></p>This is just to let you know that this use won't be notified if a breach occurs.</div>
```
Comment (roxy-dao) [inline content/docs/coldchain/notifications.md:27]:
```suggestion
<div class="tip">If the store has the vaccine module enabled, and the user does not have permission to view logs, or some other error occurs, then you will see a notification when you first login or change stores: <p><img src="/docs/coldchain/images/notification_warning.png" alt="Notification permission warning" /></p>This is just to let you know that this user won't be notified if a breach occurs.</div>
```

File: content/docs/coldchain/sensors.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+![Sensor details](/docs/coldchain/images/sensor_details.png)
+
+On this screen you can 
+- Edit the name of the sensor 
+- Assign the sensor to a location
+
+<div class="note">If the sensor is one that synchronises with Cold Chain application (e.g. the type is Blue Maestro or Laird), then the name must be edited in the Cold Chain application</div>
+
+The other details of the sensor, such as Battery level and Last recorded temperature are also shown on this screen.
```
Comment (roxy-dao) [inline content/docs/coldchain/sensors.md:72]: don't need to capitalise battery or last

**Review by roxy-dao — APPROVED** [review]
Thanks Mark~ /o/ Just a few requests but will approve for you to merge when ready (:

File: content/docs/coldchain/monitoring.md (hunk 4 above)
Comment (mark-prins) [reply content/docs/coldchain/monitoring.md:68]: It's now in v1.5.0 - coming from [this pr](https://github.com/msupply-foundation/open-msupply/pull/2439/files#diff-39160c843a1e41bf90b0c62a3b1fc4f71338b6f6487e4d4e73ff84383f05f9d6)

File: content/docs/coldchain/sensors.md (hunk 8 above)
Comment (mark-prins) [reply content/docs/coldchain/sensors.md:72]: thanks - reads better lowercase. I was taking the column headers, but should have quotes

**Pull request merged**